_topic_map.yml

```diff
@@ -47,24 +47,30 @@ Distros: openshift-dedicated
 Topics:
 - Name: Accessing your services
   File: accessing-your-services
-- Name: Cloud Infrastructure Access
-  Dir: cloud_infrastructure_access
-  Topics:
-    - Name: Accessing AWS infrastructure
-      File: dedicated-aws-access
-    - Name: Configuring AWS VPC peering
-      File: dedicated-aws-peering
-    - Name: Configuring AWS VPN
-      File: dedicated-aws-vpn
-    - Name: Configuring AWS Direct Connect
-      File: dedicated-aws-dc
 - Name: Scaling your cluster
   File: scaling-your-cluster
 - Name: Deleting your cluster
   File: deleting-your-cluster
 - Name: Networking
   File: dedicated-networking
 ---
+Name: Cloud Infrastructure Access
```

Comment:
All of these titles should be sentence case both here in the TOC and in the assemblies themselves.